src/betterproto/__init__.py

```diff
@@ -926,9 +926,10 @@ def to_dict(
         """
         output: Dict[str, Any] = {}
         field_types = self._type_hints()
+        defaults = self._betterproto.default_gen
         for field_name, meta in self._betterproto.meta_by_field_name.items():
             field_type = field_types[field_name]
```

Comment:
doesn't look like this variable has good reason to be here anymore.

Reply:
I thought it would be better here to be consistent with `field_types`

Reply:
Given the the value is no longer needed till near the end of the function, if at all, I think it's best to remove this assignment here now.

Reply:
It's used on https://github.com/Gobot1234/python-betterproto/blob/fixes/src/betterproto/__init__.py#L932 🤔

Reply:
Just to make sure we're on the same page, I'm referring to `field_type`, not `defaults`.

Reply:
I didn't add that

Reply:
Indeed. Though you changed how it's used, so it would be good to clean it up in the process.

Reply:
But wouldn't that mean the method was called multiple times in the loop when it could be called just once?

Reply:
I'm talking about `field_type = field_types[field_name] ` which as it stands after you change is defined *inside* the loop and then maybe used once (though not used at all on most code paths).